Scripts/WheatManagerC3.cs

```diff
@@ -0,0 +1,184 @@
+using System;
```

Comment:
Did it work, @MStowerUQ?

Reply:
The scripts are producing output, but I think there is some more fine tuning required before merging - the numbers still aren't quite right. Having a discussion with Alex about it, shouldn't take too long to resolve.